Controllers/Settings.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Leantime\Plugins\OmniSearch\Controllers;
+
+use Leantime\Core\Controller;
+
+/**
+ * Settings Controller for Sample Plugin
+ *
+ * @package    leantime
+ * @subpackage plugins
+ */
+class Settings extends Controller
+{
+    /**
+     * init
+     *
+     * @return void
+     */
+    public function init(): void
+    {
+    }
```

Comment:
Why are these functions empty. Please leave comment if they should be leaved empty but is required by the framework.

Reply:
Removed the controller until a settings page is needed. Doesn't seem to affect the workingness of the plugin